guide/english/php/functions/time/index.md

```diff
@@ -16,3 +16,21 @@ echo time();
 ```
 #### More Information:
 * <a href="https://secure.php.net/manual/en/function.time.php">php.net time() manual</a>
+
+## Default timezone
+
+You can set or get the settings using `date_default_timezone_set()` and `date_default_timezone_get()`.
```

Comment:
As a suggestion make it more clear to the reader, what settings you are peering here.
you can make changes as follows:-

`date_default_timezone_set()` :- Used to set the default time zone

Similarly for `date_default_timezone_get()`